contextTree.sh

```diff
@@ -44,7 +44,7 @@ function add_standard_pairs_to_stack() {
   local input_file="$1"; shift
   local output_file="$1"; shift
 
-  pushTempDir "${FUNCNAME[0]}_XXXX"
+  pushTempDir "${FUNCNAME[0]}_XXXXXXXX"
```

Comment:
Any reason this one is 8 chars where the rest are 6? I presume the alpine minimum is 6?

Reply:
Nope, must have been when I was doing the find and replace. Fixed